Mage/src/main/java/mage/game/turn/BeginCombatStep.java

```diff
@@ -38,4 +38,7 @@ public BeginCombatStep copy() {
         return new BeginCombatStep(this);
     }
 
+    public String toString() {
```

Comment:
Use override

Reply:
oh indeed, I added the abstract ones last and forgot to go back and add the attribute.